app/assets/stylesheets/tooltip.css.scss

```diff
@@ -0,0 +1,75 @@
+
+.tooltip-item {  
+  ///////////////////////////////////////////////////////////////////////////////////
+  $base-border-color: gainsboro !default;
+  $base-border-radius: 3px !default;
+  $base-line-height: 1.5em !default;
+  $dark-gray: #333 !default;
+  $base-font-color: $dark-gray !default;
+  
+  p {
+    color: $base-font-color;
+    line-height: $base-line-height;
+    margin: 0;
+  }
+  //////////////////////////////////////////////////////////////////////////////////
+
+  $tooltip-background: white;
+  $tooltip-color: $base-font-color;
+  $tooltip-max-width: 16em;
+  $tooltip-arrow-width: 8px;
+  $tooltip-shadow: 0 2px 2px silver;
+  $tooltip-distance-from-item: 3em;
+  $tooltip-arrow-distance-from-box: -1.3em;
+
+  background: $tooltip-background;
+  border-radius: $base-border-radius;
```

Comment:
Hey, Michael! Generally, you should just order things alphabetically. It's really easy to do in vim – just highlight a bunch of lines and type `:sort`.